Eatery Blue/UI/ListScreen/ListModelController.swift

```diff
@@ -41,10 +41,10 @@ class ListModelController: ListViewController {
                 }
             }
 
-            updateEateries(filteredEateries)
+            updateEateries(filteredEateries, allEateries: allEateries)
 
         } else {
-            updateEateries(allEateries)
+            updateEateries(allEateries, allEateries: allEateries)
```

Comment:
What...

Reply:
fixed. would have caused weird bug but should now working as intended. the previous naming convention in the ListModelController was weird. Even though the variable was named "allEateries" in model controller it was actually just the list eateries so i change it to be listEateries and allEateries, with the correct eateries in each respective variable